2020-epc-dev/demo-embed-arcgis/src/components/ContextPane.js

```diff
@@ -10,6 +10,12 @@ import Card, {
 } from 'calcite-react/Card';
 import {Pntm} from '../components/Common';
 
+var UserName = "";
```

Comment:
Remove these variables

As an FYI, variables should be contained and transmitted through the functions / classes that define the app not floating outside them like here. The default react "transmission" paradigm is that values get passed from the top down, so the equivalent approach here would be to have setter functions (like `ClearLogin`) live at a shared parent of `ContextPane` and the relevant app, then that shared parent can pass the values down as props. This isn't needed here, however, as the variables don't need to change and be passed around. The `Worker` app will always have the same hardcoded values.